packit/cli/sync_to_upstream.py

```diff
@@ -0,0 +1,50 @@
+"""
+Update selected component from upstream in Fedora
+"""
+
+import logging
+import os
+
+import click
+
+from packit.api import PackitAPI
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception
+from packit.config import pass_config, get_context_settings, get_local_package_config
+
+logger = logging.getLogger(__file__)
+
+
+@click.command("sync-upstream", context_settings=get_context_settings())
+@click.option(
+    "--dist-git-branch",
+    help="Target branch in dist-git to release into.",
+    default="master",
+)
+@click.option(
+    "--dist-git-path",
+    help="Path to dist-git repo to work in. "
+    "Otherwise clone the repo in a temporary directory.",
+)
+@click.option(
+    "--pr-suppressed",
```

Comment:
`--without-pr`, or even `--no-pr` is easier to type

Reply:
yeah, could be.